modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -11,20 +11,21 @@ You can add a cluster to the {mtc-full} ({mtc-short}) web console.
 
 .Prerequisites
 
-If you are using Azure snapshots to copy data:
+Optional: If you are using Azure snapshots to copy data:
 
 * You must provide the Azure resource group name when you add the source cluster.
 * The source and target clusters must be in the same Azure resource group and in the same location.
 
 .Procedure
 
 . Log in to the cluster.
-. Obtain the service account token:
+. Obtain a service account token:
 +
 [source,terminal]
 ----
-$ oc sa get-token migration-controller -n openshift-migration
+$ oc sa get-token <service_account> -n <namespace> <1>
```

Comment:
@apinnick I am curious why you don't use "migration-controller"  service account here?

Reply:
@XinRedhat 
I wasn't sure whether this was one of those cases where the user could choose any service account with cluster-admin privileges. Does the user have to use migration-controller SA?

Reply:
so far yes. you can see the doc https://docs.openshift.com/container-platform/4.6/migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.html#migration-adding-cluster-to-cam_migrating-4-2-4

Reply:
Thanks. I'll revert here.